wiki/Tournaments/OMIC/2019_4K/en.md

```diff
@@ -2,9 +2,415 @@
+- After the Qualifiers round ends there will be a distribution of players based on seed.
+  - The function of seed is for the distribution of the next round, where players are randomized based on different seed.
+  - Seed distribution based on position in the Qualifiers :
+    - 1 - 16: Top Seed
+    - 17 - 32: High Seed
+    - 33 - 48: Medium Seed
+    - 49 - 56: Low Seed
+    - Players who get the Top Seed will immediately enter the Group Distribution Stage, without having to go to the Round of 56 first.
+- **Round of 56**
+  - In this round, there will be 8 groups where each group contains **2 high seed, 2 medium seed, and 1 low seed**.
+  - The players who will qualify for the next round are the players who are in the top **2 positions** in each group.
+  - In the Round of 56 there will be **7 mappools (with 1 Tiebreaker)**.
+  - Each player collects points by reaching **positions 1 to 3**.
+  - The scoring system is as follows:
+    - 1st: 3 Point
+    - 2nd: 2 Point
+    - 3rd: 1 Point
+    - If both players get the same score in the 1-2 position, then the points will be taken from the top position. For example :
+      - Player A's score: 997.880 (3 Poin)
+      - **Player B's score: 995.522 (2 Point)**
+      - **Player C's score: 995.522 (2 Point)**
+      - Player D's score: 994.695 (0 Point)
+    - The Tiebreaker map will be used when players with **the second position and below have the same points** to get the 2nd place.
+- **Conditions to qualify from Group Distribution to Bracket Stage**
+  - Each group consists of **4 players** and the total number of players in the entire group is **32 players**.
+  - The winning condition for the Group Distribution Stage is the player who wins the first **4 points in a match**.
+  - There will be 2 out of 4 players who qualify for the next round. The final standings per group are determined from:
+    - The amount of matches won by players. If it's the same, then it will be determined from:
+    - The difference between the picks won by the player during the match. If it's still the same, then it will be determined from  **the highest Seed during Qualifiers**.
+- **Match Rules (Round of 32 until Grand Final)**
+  - If one of the players is not present, then the player who is present in the match will be declared **win automatically (*win by default*)**.
+  - Before the game starts, players will roll first. Players who win in the roll can choose **to ban** first and followed by the player who lost the roll. After both players have determined the map that is banned, the player will choose the map that is going to be played.
+  - If both players got the same score, then **the match will be replayed using the same map**.
+  - The Tiebreaker map is played when both players get a difference of one in point with the winning points that **should have gotten** to **win the match**.
```

Comment:
I am not sure what this means? Shouldn't TB's be played when they have the same amount of map points?

Reply:
indeed, does "both players get a difference of one in point with the winning points" sounds too ambiguous?

Reply:
I think so, should be something along the lines of 
```suggestion
  - The Tiebreaker map is played when both players have the same amount of points and 1 more point is required to **win the match** by either player.
```

Reply:
Right then, sure